Frontend/ui-customer/src/Screens/CustomerUtils/AddNewAddressScreen/ComponentConfig.purs

```diff
@@ -44,7 +44,7 @@ primaryButtonConfigConfirmLoc state = let
     , background = Color.black900
     , margin = (Margin 0 22 0 16)
     , id = "AddNewaddressConfirmLocationButton"
-    , isClickable = state.props.isServiceable
+    -- , isClickable = state.props.isServiceable
```

Comment:
instead of this can we make button disabled and color fade for button when lat/lng is not serviceable

Reply:
but still user won't get the context that it is disabled because it is unserviceable. Instead if they click on confirm then they will get the location unserviceable screen.

Reply:
okay we can ask designer for this later, as this would make thing easier for user